packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1056,6 +1056,9 @@ mixin TextSelectionDelegate {
   /// Whether search web is enabled, must not be null.
   bool get searchWebEnabled => true;
 
+  /// Whether share is enabled, must not be null.
+  bool get shareEnabled => true;
```

Comment:
what does "must not be null" mean? the `bool` is not nullable

Reply:
Good question lol, the comment is just from me copying the cut/copy/paste fields. Maybe bool used to be nullable? Ill remove it anyways.

Reply:
Nit: Can you remove it from the other fields above too? (up to cutEnabled)

These comments existed before null safety, and even until recently we still supported non-null-safe dart, so the comments were relevant to those users.  Not anymore though!